context/app/static/js/components/bulkDownload/BulkDownloadDialog/BulkDownloadDialog.tsx

```diff
@@ -0,0 +1,224 @@
+          the entire process. */}
+        </Box>
+        <RelevantPagesSection pages={PAGES} />
+      </Stack>
+    </SectionDescription>
+  );
+}
+
+interface ProtectedDatasetsSectionProps {
+  control: Control<BulkDownloadFormTypes>;
+  errorMessages: string[];
+  protectedHubmapIds: string;
+  protectedRows: DatasetAccessLevelHits;
+  removeProtectedDatasets: () => void;
+}
+function ProtectedDatasetsSection({
+  control,
+  errorMessages,
+  protectedHubmapIds,
+  protectedRows,
+  removeProtectedDatasets,
+}: ProtectedDatasetsSectionProps) {
+  if (errorMessages.length === 0) {
+    return null;
+  }
+
+  return (
+    <Stack paddingY={1}>
+      <ErrorOrWarningMessages errorMessages={errorMessages} />
+      <RemoveProtectedDatasetsFormField
+        control={control}
+        protectedHubmapIds={protectedHubmapIds}
+        removeProtectedDatasets={removeProtectedDatasets}
+        protectedRows={protectedRows}
+      />
+    </Stack>
+  );
+}
+
+function DownloadOptionsDescription() {
+  return (
+    <SectionDescription>
+      <Stack spacing={2}>
+        <Box>Select raw and/or processed files to download.</Box>
+        <LabelledSectionText label="Raw Data" spacing={1}>
+          Raw data consists of files as originally submitted by the data submitters. Individuals files for raw data
+          cannot be previewed or selected for the manifest download. You must download all raw files, and the total
+          download size cannot be estimated.
+        </LabelledSectionText>
+        <LabelledSectionText label="Processed Data" spacing={1}>
+          Processed data includes files associated with data generated by HuBMAP using uniform processing pipelines or
+          by an external processing approach. Only files centrally processed by HuBMAP are available for individual
+          selection, which can be done in the Advanced Selections section below.
+        </LabelledSectionText>
+      </Stack>
+    </SectionDescription>
+  );
+}
+
+interface DownloadOptionsSectionProps {
+  control: Control<BulkDownloadFormTypes>;
+  downloadOptions: {
+    key: string;
+    label: string;
+  }[];
+  isLoading: boolean;
+  errorMessages: string[];
+  protectedHubmapIds: string;
+  protectedRows: DatasetAccessLevelHits;
+  removeProtectedDatasets: () => void;
+}
+function DownloadOptionsSection({
+  control,
+  downloadOptions,
+  isLoading,
+  errorMessages,
+  protectedHubmapIds,
+  protectedRows,
+  removeProtectedDatasets,
+}: DownloadOptionsSectionProps) {
+  if (isLoading) {
+    return (
+      <>
+        <Skeleton variant="rectangular" height={50} />
+        <Skeleton variant="rectangular" height={300} />
+        <Skeleton variant="rectangular" height={200} />
+      </>
+    );
+  }
+
+  if (downloadOptions.length === 0) {
+    return (
+      <Box paddingTop={1}>
+        <Alert severity="warning">
+          <Typography>Files are not available for any of the selected datasets.</Typography>
+        </Alert>
+      </Box>
```

Comment:
Do we need the `Box` is the parent a flex container?

Reply:
The `Box` is needed to prevent the `spacing` style from the parent `Stack` from applying to the children of the `Step` component.

Reply:
The `Box` was to apply the `paddingTop` styling somewhere since `sx` doesn't work on `Alerts` - however it looks like the padding is actually unnecessary here, so I'll remove it.